### ISSUE TITLE
Supplementary periodic tables colormaps

### ISSUE THREAD

**Issue by eimrek** [aiidateam/acwf-verification-scripts#35]
Hi,

#34 makes the maximum value of the colorbars for each code consistent for the supplementary periodic tables.

I could also update the colormap, but the choice is not very clear to me.

In [https://acwf-verification.materialscloud.org/](https://acwf-verification.materialscloud.org/), my colormap is the following: 

```
0.0 - dark green
excellent_match - light-green
good_match - white
3*good_match - red
```

with linear interpolation inside and constant extrapolation (e.g. something that is `10*good_match` is still the same red). In that app, this is not an issue, as I also have the exact numbers in the plotted matrix.

However, in these supplementary periodic table, it might make sense to have a colormap that captures the full range.

So perhaps something like the following might make sense:

```
0.0 - dark green
excellent_match - light-green
good_match - white
3*good_match - red
maximum_value - some-other-color
```

with linear interpolation between them. Does this make sense? Or should we do something else? Pinging @giovannipizzi @bosonie

**Comment by eimrek** [comment]
@zhubonan thanks for noticing. fixed.

**Comment by eimrek** [comment]
closing this issue, as it was resolved via #34
